src/DataManagement/Saving/AlternativeSavingCalculator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using ePiggyWeb.DataManagement.Entries;
+using ePiggyWeb.DataManagement.Goals;
+using ePiggyWeb.Utilities;
+
+namespace ePiggyWeb.DataManagement.Saving
+{
+    public static class AlternativeSavingCalculator
```

Comment:
As I understand this does the same and just code improvements?

Reply:
Yeah pretty much